plugins/filter_apm_kubernetes_labels/filter_kubernetes_labels.c

```diff
@@ -228,13 +256,14 @@ static int cb_modifier_filter_apm_kubernetes_labels(const void *data, size_t byt
         int key_len;
         int val_len;
         bool pod_name_matched = false;
-        for (i = 1; i < ctx->jsmn_ret; i++) {
+        for (i = 1; ((i < ctx->jsmn_ret) && (flag)); i++) {
             t = &ctx->jsmn_tokens[i];
             if (t->type != JSMN_STRING) {
                 continue;
             }
 
             if (t->start == -1 || t->end == -1 || (t->start == 0 && t->end == 0)){
+                flb_free(t);
```

Comment:
dont free this variable, its just pointing to ctx->jsmn_tokens, which never should be freed